connector/src/main/java/org/geysermc/connector/utils/MessageUtils.java

```diff
@@ -94,6 +100,25 @@ public static String getBedrockMessage(Message message) {
         return builder.toString();
     }
 
+    public static String getBedrockMessageWithTranslate(Message message) {
+        return getBedrockMessageWithTranslate(message, null);
+    }
+
+    private static String getLocaleString(String messageText, String locale) {
+        HashMap<String, String> localeStrings = Toolbox.LOCALE_MAPPINGS.get(locale.toLowerCase());
```

Comment:
Should be `Map<String, String> localeStrings = Toolbox.LOCALE_MAPPINGS.get(locale.toLowerCase())`;